tooling/eclipse-formatter-config.xml

```diff
@@ -267,5 +267,6 @@
         <setting id="org.eclipse.jdt.core.formatter.enabling_tag" value="CHECKSTYLE:ON"/>
         <setting id="org.eclipse.jdt.core.formatter.comment.format_block_comments" value="false"/>
         <setting id="org.eclipse.jdt.core.formatter.comment.format_line_comments" value="false"/>
+        <setting id="org.eclipse.jdt.core.formatter.comment.format_javadoc_comments" value="false"/>
```

Comment:
What problem is this solving?

Reply:
The formatter plugin does not keep aliment with parameter which hurts readability

Reply:
Could you please try if the following combination does what you want?

```
<setting id="org.eclipse.jdt.core.formatter.comment.format_javadoc_comments" value="true"/>
<setting id="org.eclipse.jdt.core.formatter.comment.align_tags_names_descriptions" value="true"/>
<setting id="org.eclipse.jdt.core.formatter.comment.indent_root_tags" value="false"/>
<setting id="org.eclipse.jdt.core.formatter.comment.align_tags_descriptions_grouped" value="false"/>
```

Reply:
would you mind sending a follow up pr ? I don't want to waste another build cycle and I'm getting a little tired of this auto formatter thing